pkgs/applications/networking/instant-messengers/neosay/default.nix

```diff
@@ -0,0 +1,27 @@
+{ lib
+, buildGoModule
+, fetchFromGitHub
+}:
+
+buildGoModule rec {
+  pname = "neosay";
+  version = "1.0.0";
+
+  src = fetchFromGitHub {
+    owner = "donuts-are-good";
+    repo = "neosay";
+    rev = "v${version}";
+    hash = "sha256-Uwz6Y26AtzWXLFgJY0WVD0HBb+vbMeeMKt8gCk6viec=";
+  };
+
+  vendorHash = "sha256-w0aZnel5Obq73UXcG9wmO9t/7qQTE8ru656u349cvzQ=";
+
+  ldflags = [ "-s" "-w" ];
+
+  meta = with lib; {
+    description = "Pipe stdin to matrix";
+    homepage = "https://github.com/donuts-are-good/neosay";
+    license = licenses.unfree;
```

Comment:
```suggestion
    license = licenses.mit;
```

Reply:
> `neosay` is licensed under the MIT license. Check out the LICENSE file for more info. If you dont get it don't worry about it.

lol

Reply:
What do they mean they don't have a license file :cry: 
I probably should have waited with opening this PR until I'm not sleep deprived :zzz: 
Anyways I opend a issue telling them about the missing license file, https://github.com/donuts-are-good/neosay/issues/4

Reply:
> Anyways I opend a issue telling them about the missing license file

me too.... donuts-are-good/neosay#3

Reply:
Wasn't there when I started writing mine, we opened them with just a minute difference.

Reply:
fixed